File: app/src/main/java/life/plank/juna/zone/data/network/model/FootballTeam.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class FootballTeam {
    private Integer id;
    private Integer foreignId;
    private String name;
    private Boolean isNationalTeam;
    private Integer founded;
    private String logoLink;
}

File: app/src/main/java/life/plank/juna/zone/data/model/FootballTeam.kt
package life.plank.juna.zone.data.model

import android.os.Parcelable
import androidx.room.Entity
import kotlinx.android.parcel.Parcelize

@Entity
@Parcelize
data class FootballTeam(
        var id: Int?,
        var foreignId: Int?,
        var name: String?,
        var isNationalTeam: Boolean?,
        var founded: Int?,
        var logoLink: String
) : Parcelable